packages/cozy-scanner/src/EditDocumentQualification.jsx

```diff
@@ -37,34 +45,10 @@ class EditDocumentQualification extends Component {
     const theme = item ? getThemeByItem(item) : null
     const categoryLabel = item ? theme.label : null
     return (
-      <ExperimentalModal
-        title={document.name}
-        dismissAction={onClose}
-        primaryText={t('Scan.apply')}
-        primaryAction={async () => {
-          if (isOffline) {
-            Alerter.error(t('Scan.error.offline'))
-          } else {
-            try {
-              const fileCollection = client.collection('io.cozy.files')
-              const updatedFile = await fileCollection.updateMetadataAttribute(
-                document._id,
-                qualification
-              )
-              pushAnalytics(item)
-              if (onDescribed) onDescribed(updatedFile.data)
-              onClose()
-              Alerter.success(t('Scan.successful.qualified_ok'))
-            } catch (error) {
-              Alerter.error(t('Scan.error.generic'))
-            }
-          }
-        }}
-        primaryType={'regular'}
-        secondaryText={t('Scan.cancel')}
-        secondaryAction={() => onClose()}
-        secondaryType={'secondary'}
-        description={
+      <ExperimentalDialog>
```

Comment:
You may need an `onClose` here as well so that the dialog closes when we click the overlay behind it.